src/main/java/frc/trigon/robot/RobotContainer.java

```diff
@@ -6,8 +6,17 @@
 package frc.trigon.robot;
 
 import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import frc.trigon.robot.subsystems.arm.Arm;
+import frc.trigon.robot.subsystems.arm.ArmCommands;
+import frc.trigon.robot.subsystems.arm.ArmConstants;
+import frc.trigon.robot.subsystems.roller.Roller;
 
 public class RobotContainer {
```

Comment:
Remember for next time to not commit changes to RobotContainer unless they're changes that are going to be on the real robot

Reply:
I'll fix it when I am done with the arm sim

Reply:
It's fine for now
 Just for the future, no need to delete it, but don't check the commit box when sending a commit